__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -1,3 +1,3 @@
+exports[`library consistently exports 1`] = `"on :: function once :: function off :: function emit :: function emitter :: function eventListeners :: object calls :: object createMiddleware :: function createMiddlewareContext :: function createMiddlewareFactory :: function TYPES :: object ENUMS :: object expectedType :: function expectedArrayOfType :: function enumerableException :: function unknownKey :: function unknownEventName :: function validateInput :: function getCacheOptions :: function OneServiceWorkerError :: function errorFactory :: function notEnabled :: function notSupported :: function failedToInstall :: function failure :: function configure :: function getConfig :: function isDevelopment :: function isEventsEnabled :: function isNonStandardEnabled :: function isNavigationPreloadEnabled :: function isServiceWorker :: function isServiceWorkerSupported :: function isNotificationSupported :: function isPushSupported :: function isBackgroundSyncSupported :: function isCacheStorageSupported :: function isIndexedDBSupported :: function isPermissionsSupported :: function isOffline :: function getRegistration :: function getRegistrations :: function register :: function getWorker :: function postMessage :: function update :: function unregister :: function escapeHatch :: function showNotification :: function getNotifications :: function getSubscription :: function subscribe :: function unsubscribe :: function registerTag :: function getTags :: function urlBase64ToUint8Array :: function cachePrefix :: string cacheDelimiter :: string defaultCacheName :: string defaultCacheOptions :: object createCacheName :: function normalizeRequest :: function open :: function has :: function keys :: function match :: function matchAll :: function add :: function addAll :: function put :: function remove :: function removeAll :: function entries :: function clear :: function metaDataCacheName :: string createMetaCacheName :: function createMetaCacheEntryName :: function createMetaRequest :: function createMetaResponse :: function getMetaStore :: function getMetaData :: function setMetaData :: function deleteMetaData :: function createCacheEntryName :: function onInstall :: function onActivate :: function onMessage :: function onPush :: function onSync :: function onFetch :: function cacheBusting :: function cacheRouter :: function cacheStrategy :: function clientsClaim :: function escapeHatchRoute :: function manifest :: function messageContext :: function messenger :: function navigationPreloadActivation :: function navigationPreloadResponse :: function appShell :: function precache :: function skipWaiting :: function expiration :: function"`;
```

Comment:
Shouldn't our snapshots be somewhat readable and reasonably sized. At best this is a smoke test, meaning it will tell us something is wrong however it will not tell us what in practice. Can we find a way to make this more readable and sized appropriate? For example, if this captures a timeline of events, turn this snapshot into an array of actions that should happen in a specific order.

Reply:
For the `index.spec.js` snapshot, it's not actually needed and is used to see changes in what the library is ultimately exporting out - not related to functionality at all. We can delete it altogether however it is useful to see any changes made in the API surface area.

Reply:
Updated 52d85e6

Reply:
👍